main.tf

```diff
@@ -1,14 +1,18 @@
 terraform {
   required_providers {
     http = {
-      source  = "hashicorp/http"
+      source = "hashicorp/http"
     }
     local = {
-      source  = "hashicorp/local"
+      source = "hashicorp/local"
     }
   }
 }
 
+variable "opsgenie_api_key_cluster_monitoring" {
+  description = "API Key of Heartbeat created"
```

Comment:
I would describe this as: "OpsGenie API Key to send heartbeats"

Reply:
that name works for me